english/make-friends/where_am_i

```diff
@@ -0,0 +1,16 @@
+You realise you need as much information as you need to get out of this situation.
```

Comment:
It looks like there is no link to this new file.  Did you forget to commit that change?  Make sure you create (and commit) an option for each new "page" you create.